_templates/new/help/index.ejs.t

```diff
@@ -0,0 +1,5 @@
+---
+message: |
+  - hygen new {bold component} --name [NAME] --path [PATH] (default: components) --functional (default: false)
```

Comment:
I think you removed the `functional` option.

Reply:
Thanks >_<, I've forgot to remove this option 5f01c3c

I've first implemented the component with a `functional` option, but later on I've checked and we don't have any functional component yet.